corehq/apps/users/models.py

```diff
@@ -763,7 +763,7 @@ class CouchUser(Document, DjangoUserMixin, IsMemberOfMixin, UnicodeMixIn, EulaMi
     has_built_app = BooleanProperty(default=False)
 
     _user = None
-    _user_checked = False
```

Comment:
`_user_checked` isn't referenced anywhere (and as far as I can see never has been)